### ISSUE TITLE
The trainer class of the medkit library lack of more specific options

### ISSUE THREAD

**Issue by Thibeb** [medkit-lib/medkit#19]
When fine-tuning a BERT model using the medkit library I had to compare it to a fine-tuning process made [directly with the hugging face library](https://medium.com/@yanis.labrak/how-to-train-drbert-on-french-biomedical-named-entity-recognition-ner-using-huggingface-and-c84c91456d56) and I stood upon the fact that I couldn't replicate exactly what was done with the HF library in medkit.

Options such as weight decay and or early stopping could not be implemented.

I don't know if it would be possible to have more access to the hugging face training library directly trough the medkit-lib but it would help having more options when training models.
